### ISSUE TITLE
Problem in get_reviewable_task_runs() about jsonify DictConfig 

### ISSUE THREAD

**Issue by wayne163** [facebookresearch/Mephisto#536]
I came with a problem when using client scripts. I think maybe it's a type conflict bug. 
Version: 0.3.4
Steps
1. start app with ```python cli.py web```
2. send get request to ```http://localhost:5000/api/v1/task_runs/reviewable``` and ten flask will throw error
```
[2021-09-01 12:54:36,582] ERROR in app: Exception on /api/v1/task_runs/reviewable [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Mephisto/mephisto/client/api.py", line 68, in get_reviewable_task_runs
    return jsonify({"task_runs": dict_tasks, "total_reviewable": reviewable_count})
  File "/usr/local/lib/python3.8/site-packages/flask/json/__init__.py", line 370, in jsonify
    dumps(data, indent=indent, separators=separators) + "\n",
  File "/usr/local/lib/python3.8/site-packages/flask/json/__init__.py", line 211, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/usr/local/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.8/site-packages/flask/json/__init__.py", line 99, in default
    return text_type(o.__html__())
TypeError: 'NoneType' object is not callable
```
3. I found the problem about ```params``` key in task_runs. In TaskRun [to_dict()](https://github.com/facebookresearch/Mephisto/blob/v0.3.4/mephisto/data_model/task_run.py#L323), 'params' key will be set to ```self.get_task_config().args``` which is [TaskConfig.args](https://github.com/facebookresearch/Mephisto/blob/3665e05088ffa8b556d7d1228cc8542670e076b8/mephisto/data_model/task_config.py#L111) as Dict[str: any] type, 
```
self.db = task_run.db
args = task_run.args
if args is None:
      return

# Parse out specific arguments for the task_config
self.args: Dict[str, Any] = args.task
```
but the [task_run.args](https://github.com/facebookresearch/Mephisto/blob/3665e05088ffa8b556d7d1228cc8542670e076b8/mephisto/data_model/task_run.py#L54) is actually DictConfig type, so the ```params``` is also DictConfig type not Dict.
```
try:
     self.args: "DictConfig" = OmegaConf.create(json.loads(self.param_string))
except Exception as e:
     self.args = None
     print(e)
```
Now I temporarily used OmegaConf.to_container to parse the ```params``` data before jsonify.

**Comment by JackUrb** [comment]
Thanks for the find and resolution here! Indeed the correct behavior is to use `to_container` inside of the `to_dict` method. Happy to accept a PR for this fix (or will implement it if you prefer).

**Comment by wayne163** [comment]
Here is the [PR](https://github.com/facebookresearch/Mephisto/pull/547) @JackUrb

**Comment by JackUrb** [comment]
Resolved in #547
